MosaicSlider/MosaicSlider.js

```diff
@@ -68,16 +85,19 @@ export const sketch = (p) => {
   };
 
   p.mousePressed = () => {
+    SOUND.init();
```

Comment:
This doesn't work in mobile. pointer down events for mobile devices don't count for user activation (see [MDN](https://developer.mozilla.org/en-US/docs/Web/Security/User_activation)). The point seems to be to prevent page scrolling from activating annoying sounds/popups. (see https://github.com/whatwg/html/issues/7341)

To address this, let's do this:

1. Make some sort of Play button overlay that initializes the audio and disappears when clicked (using the appropriate event(s) for desktop vs mobile)